tests/baselines/reference/declarationEmitGlobalThisPreserved.js

```diff
@@ -168,125 +168,3 @@ export declare class AsClassProperty {
     isNaN?: typeof globalThis.isNaN;
 }
 export type AsFunctionType = (isNaN: typeof globalThis.isNaN) => typeof globalThis.isNaN;
-
-
-//// [DtsFileErrors]

```

Comment:
Note the the errors going away here; I constructed these tests such that they will emit a d.ts error when they're wrong, rather than us having to figure out if the output looks right or not.